copy_edit.py

```diff
@@ -82,17 +82,29 @@ def run(self, edit):
 		else:
 			strs_per_sel = numstrings
 		
-		str_index = 0
 		new_sels = []
-		for sel in self.view.sel():
```

Comment:
What if it was just `for sel in reversed(self.view.sel()):`? Would that fix it? That way you're replacing from the end. Or maybe I'm misunderstanding the bug?

Reply:
Please specify which commit your question is related to.

Reply:
Ok, I've found the related commit, but I can not I cannot think of how it can be reimplemented (simple adding `reverse` is not enough because of `str_index` usage inside that loop).
You can play with the code yourself if you wish, but do not forget to run tests (your suggestion break test `(1)`).